Chinese/Casey/Casey.md

```diff
@@ -1,3 +1,4 @@
 Casey现在为udacity工作，自动合并GitHub的pull request。
 Casey is a help bot.
-Casey won't accept a pull request that a fix typo.
\ No newline at end of file
+Casey won't accept a pull request that a fix typo.
+直到有一天Casey收到来自转行开始学编程的YouTuber [CaseyNeistat](https://github.com/crossairplane/create-your-own-adventure/blob/neistat/Chinese/Casey/neistat/neistat.md)的pull request.
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.